### PR TITLE
Nav Bar fix & Other UI stuff

### PR DESCRIPTION
**Pull request by schnajos**

### CODE REVIEW

File: frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -27,9 +27,9 @@ const routes: Routes = [
   { path: 'my-products', component: MyProductsComponent, canActivate: [AuthGuard] },
   { path: 'my-products/new', component: CreateProductComponent, canActivate: [AuthGuard] },
   { path: 'product/:id/edit', component: EditProductComponent, canActivate: [AuthGuard] },
-  { path: 'admin/dashboard', component: AdminPanelComponent, canActivate: [AdminGuard] },
+  { path: 'admin/dashboard', component: AdminPanelComponent, canActivate: [AdminGuard]  },
   { path: 'admin/dashboard/users', component: AdminUserPanelComponent, canActivate: [AdminGuard] },
-  { path: 'admin/dashboard/products', component: AdminProductPanelComponent, canActivate: [AdminGuard] },
+  { path: 'admin/dashboard/products', component: AdminProductPanelComponent, canActivate: [AdminGuard]  },
```
Comment (KnownInter) [inline frontend/src/app/app-routing.module.ts:32]: Why did you add the additional spaces at the end?

File: frontend/src/app/components/create-product/create-product.component.spec.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     TestBed.configureTestingModule({
-      declarations: [ CreateProductComponent ]
+      declarations: [ CreateListingComponent ]
     })
     .compileComponents();
   }));
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(CreateProductComponent);
+    fixture = TestBed.createComponent(CreateListingComponent);
```
Comment (KnownInter) [inline frontend/src/app/components/create-product/create-product.component.spec.ts:17]: It should be CreateProductComponent and not CreateListingComponent.

File: frontend/src/app/components/edit-product/edit-product.component.html [hunk 3]
```diff
@@ -37,7 +37,7 @@ <h1>Edit Product '{{product.title}} #{{product.productId}}'</h1>
     </mat-form-field>
     <mat-form-field appearance="outline" class="sell-or-lend">
       <mat-label>Sell or Lend?</mat-label>
-      <mat-select formControlName="sellOrLend" [disabled]="productForm.value.type == 1 || productForm.value.priceKind == 1 || productForm.value.priceKind == 2">
```
Comment (KnownInter) [inline frontend/src/app/components/edit-product/edit-product.component.html:40]: Those conditions were added for a more dynamic formular. Why did you remove them?

File: frontend/src/app/components/my-products/my-products.component.css [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
+  margin: 4% auto 40px;
+  width: 60%;
+  padding: 25px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(6, minmax(0, 1fr));
+  grid-column-gap: 15px;
+}
```
Comment (KnownInter) [inline frontend/src/app/components/my-products/my-products.component.css:12]: I don't see any usage of this.

File: frontend/src/app/components/profile/profile.component.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    margin: 4% auto 40px;
+    width: 60%;
+    padding: 25px;
+  }
+  
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(6, minmax(0, 1fr));
+    grid-column-gap: 15px;
+  }
```
Comment (KnownInter) [inline frontend/src/app/components/profile/profile.component.css:12]: You don't need to add a grid everywhere if you don't need it.

**Review by KnownInter — CHANGES_REQUESTED** [review]
There seems to be a couple of issues:
- The logo doesn't show up
- Hovering the icons makes them invisible
- The 'Admin Dashboard' Icon is completely invisible
- Be consistent with product or listing
- The navigation bar doesn't scale properly with smaller screens
- I feel like the navigation bar takes up too much space with that height
- There are sometimes random overflows eventhough the complete height isn't in use

Overall it looks a bit too whitish. Maybe add different types of white to add some variety. And as we've discussed before, there is no need to force the design of the mockups. We can use the mockups as an orientation, so don't waste too much time implementing it. Also create a new branch for the mockups, so we can just merge it into master.

File: frontend/src/app/app-routing.module.ts (hunk 1 above)
Comment (schnajos) [reply frontend/src/app/app-routing.module.ts:32]: Oh oops, that's a mistake from pasting the adminGuard back in after disabling it 😅

File: frontend/src/app/components/profile/profile.component.css (hunk 5 above)
Comment (schnajos) [reply frontend/src/app/components/profile/profile.component.css:12]: I didn't use it then but the page will make use of a grid, once I have all the elements figured out

File: frontend/src/app/components/my-products/my-products.component.css (hunk 4 above)
Comment (schnajos) [reply frontend/src/app/components/my-products/my-products.component.css:12]: In future implementation, add other views that use grid

**Pull request merged**